src/test/specs/generic/network/address/NetAddress.spec.js

```diff
@@ -174,5 +174,7 @@ describe('NetAddress', () => {
         expect(NetUtils.isPrivateIP('fd00:3456:789a:1::1')).toEqual(true);
         expect(NetUtils.isPrivateIP('fe00:3456:789a:1::1')).toEqual(false);
         expect(NetUtils.isPrivateIP('ff02:3456:789a:1::1')).toEqual(false);
+
+        expect(() => NetUtils.isPrivateIP('not-an-ip')).toThrow("Malformed IP address not-an-ip");
```

Comment:
It would be more consistent if you change the double quotes to single quotes around `Malformed IP address not-an-ip` (and since you're at it, could you do it for lines 119 and 120 too? 😉 ).

Also, this check should probably go into its own "rejects invalid private IP addresses" spec.